alfresco-transform-core-aio/alfresco-transform-core-aio-boot/src/main/java/org/alfresco/transformer/AIOController.java

```diff
@@ -26,59 +26,32 @@
  */
 package org.alfresco.transformer;
 
-import static org.alfresco.transform.client.model.Mimetype.MIMETYPE_HTML;

```

Comment:
Minor: IDE settings ? We should not reformat imports in the same PR. 

Ideally, re-format only changes are a separate PR.

Reply:
Erik asked me to do the Intellij automatic removal of unused imports. Depending on what the order was before, the used ones move.